src/__tests__/index.js

```diff
@@ -138,6 +138,16 @@ describe("Opentracing interface", () => {
       expect(zipkinTracer.recordBinary).not.toHaveBeenCalled();
     });
 
+    it("should expose the underlying tracer", () => {
```

Comment:
Can you explain why this should be exposed? Same for the context

Reply:
see comment below about the `OpenTracing` inheritance pattern...

Reply:
using the predefined methods of the base class: https://github.com/opentracing/opentracing-javascript/blob/d2c1c6a5a50439bd769d02c7f4638293db40140a/src/span.ts#L17-L33